teachopencadd/talktorials/T022_ligand_based_screening_neural_network/talktorial.ipynb

```diff
@@ -362,7 +362,7 @@
   },
```

Comment:
<p>As mentioned before, add space in both GetMorganGenerator cases after radius parameter, thank you!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/348/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>